docs/specifications/tests/Consistency-TP/consistency01.md

```diff
@@ -1,52 +1,82 @@
-## CONSISTENCY01: SOA serial number consistency
+# CONSISTENCY01: SOA serial number consistency
 
-### Test case identifier
+## Test case identifier
 
-**CONSISTENCY01:** SOA serial number consistency
+**CONSISTENCY01**
 
-### Objective
+## Objective
 
 For the data served by the authoritative name servers for a designated zone
 to be consistent, all authoritative name servers must serve the same SOA
 record for the designated zone.   
 
-If the serial number (explained in 3.3.13. of [RFC 1035](https://tools.ietf.org/html/rfc1035)) 
-, which is part of the SOA record, is not consistent between authoritative servers, 
+If the serial number (explained in 3.3.13. of [RFC 1035]), 
+which is part of the SOA record, is not consistent between authoritative servers, 
 there is a possibility that the data served is inconsistent. The reasons for this 
 inconsistency may be different - such as misconfiguration, or as a result of slow 
 propagation to the secondary name servers.
 
 The objective of this test is to verify that the serial number is consistent
 between different authoritative name servers.
 
-For reference purposes : [RFC1982](https://tools.ietf.org/html/rfc1982) 
+For reference purposes : [RFC 1982]
 explains the serial number arithmetic, and section 4.3.5 of 
-[RFC 1034](https://tools.ietf.org/html/rfc1035) explains the importance of
+[RFC 1034] explains the importance of
 serial number consistency.
 
-### Inputs
+## Inputs
 
-The domain name to be tested.
+* The domain name to be tested ("child zone")
+* Accepted SOA serial difference, default 0 ("accepted serial difference")
 
-### Ordered description of steps to be taken to execute the test case
+## Ordered description of steps to be taken to execute the test case
 
- 1. Obtain the list of name servers from [Method4] and [Method5].
- 2. Retrieve the SOA RR from all the name servers.
- 3. If the SOA serial number is not the same from all the answers received
-    from step 2, then:
-     1. Check if the difference is minor (An integer should be added to
-        define what is the minor value).
-     2. If the difference is greater than the minor value, then the test
-        case fails.
-     3. If the difference is less than or equal to the minor value,
-        then the test is passed.
+ 1. Obtain the list of name server IPs for the *child zone* from [Method4] 
+    and [Method5].
+ 2. Create an SOA query for *child zone* apex and send it to all name 
+    server IPs.
+ 3. Retrieve the SOA RR from the responses from all name server IPs.
+ 4. If a name server does not respond, emit *NO_RESPONSE*.
+ 5. If a name server does not include a SOA record in the response,
+    emit *NO_RESPONSE_SOA_QUERY*.
```

Comment:
In my view a better name for this would be NO_ANSWER_SOA_RR_SET, but maybe it's not worth changing the name.

Reply:
It is not bad enough to be worth changing it. It make sense at least.